rllib/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -105,17 +105,17 @@ def __init__(self, algo_class=None):
         self.sgd_minibatch_size = 128
         self.shuffle_sequences = True
         self.num_sgd_iter = 30
-        self.learning_starts = 1000
         self.replay_buffer_config = {
             "type": "ReplayBuffer",
             # Size of the replay buffer in batches (not timesteps!).
             "capacity": 1000,
-            # When to start returning samples (in batches, not timesteps!).
-            "learning_starts": 500,
             # Choosing `fragments` here makes it so that the buffer stores entire
             # batches, instead of sequences, episodes or timesteps.
             "storage_unit": "fragments",
         }
+        # Number of timesteps to collect from rollout workers before we start
+        # sampling from replay buffers for learning.
+        self.num_steps_sampled_before_learning_starts = 1000
```

Comment:
I don't wanna wreck this for you @ArturNiederfahrenhorst 🥲 but is this number of env steps / agent steps?

Reply:
Ok it only needs a bit more explicit documentation.

Reply:
It depends on how you count, just like with STEPS_TRAINED_COUNTER and so on. We generally refer to num_steps_*** as whatever user has chosen to count in.

Reply:
I've changed these comments.